### PR TITLE
Added creature tags and read aloud insets

### PR DESCRIPTION
**Pull request by felipeogarcia**

### CODE REVIEW

**Comment by revilowaldow** [comment]
I was literally part way through doing this 😂😭

**Comment by revilowaldow** [comment]
I hadn't gotten super far yet but it looks like I got a number of things you haven't so not wasted time!
I'll pull your latest copy and :keep going:.

image

**Comment by felipeogarcia** [comment]
> I hadn't gotten super far yet but it looks like I got a number of things you haven't so not wasted time!
> I'll pull your latest copy and :keep going:.
> 
> image

Awesome! I was more focused on a missing beast and the read aloud insets. There's definitely several places I haven't touch. Thanks for contributing!

File: data/bestiary/bestiary-ttp.json [hunk 1]
```diff
@@ -393,6 +393,90 @@
 			],
 			"hasFluff": true,
 			"hasFluffImages": true
+		},
```
Comment (TheGiddyLimit) [inline data/bestiary/bestiary-ttp.json:396]: This should be a `_copy` creature based on the MM `Plesiosaurus`

File: data/bestiary/bestiary-ttp.json [hunk 2]
```diff
@@ -393,6 +393,90 @@
...
+				"Two Heads"
+			],
+			"damageTags": [
+				"P"
+			],
+			"miscTags": [
+				"MW",
+				"RCH"
+			],
+			"hasFluff": true,
```
Comment (TheGiddyLimit) [inline data/bestiary/bestiary-ttp.json:478]: Does it?

File: data/bestiary/bestiary-ttp.json (hunk 1 above)
Comment (revilowaldow) [reply data/bestiary/bestiary-ttp.json:396]: Query on why the `Giant Slug` above it doesn't follow the `_copy` syntax by the same logic?

image

File: data/bestiary/bestiary-ttp.json (hunk 2 above)
Comment (felipeogarcia) [reply data/bestiary/bestiary-ttp.json:476]: My bad. Fixed it.

**Comment by revilowaldow** [comment]
I pulled my overhaul of the adventure in #345 
It won't be compatible with this due to differences in tagging and how the Two-Headed Plesiosaurus is added.
I did however manually merge the work done here into it where appropriate.

**Comment by felipeogarcia** [comment]
Closing this to avoid issues with #345

**Pull request closed without merging**